_data/navbars/calico-for-windows.yml

```diff
@@ -0,0 +1,31 @@
+title: Calico for Windows
```

Comment:
I think the title should be `Windows`.

Reply:
Fixed (https://github.com/projectcalico/calico/pull/4399/commits/068a20f466bcbf4cb3c9e5508a7bd998ceade0a4)